systemd/systemd.go

```diff
@@ -164,6 +169,27 @@ func NewCollector(logger log.Logger) (*Collector, error) {
 		"Resident memory size in bytes.",
 		[]string{"name"}, nil,
 	)
+
+	ipIngressBytes := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "service_ip_ingress_bytes_total"),
```

Comment:
```suggestion
		prometheus.BuildFQName(namespace, "", "service_ip_ingress_bytes"),
```

We didn't need to add `_total` to `bytes` as bytes is a unit ;)